### DISCUSSION TITLE
Yubikey PIV support status?

### DISCUSSION THREAD

**Discussion by myyra** [FiloSottile/age#380] [General]
It's been almost a year since the [mention](https://github.com/FiloSottile/age/issues/142#issuecomment-770494415) of PIV support coming "soon." So what is the status of it currently?

I'm trying to get rid of GPG, and having support for keys on multiple Yubikeys and encrypting for them would make this the ultimate tool for all encryption tasks.

**Comment by str4d** [comment]
It's here! https://github.com/str4d/age-plugin-yubikey

The thing that is missing from this repo is plugin support, which will be in the 1.1 release IIRC. In the meantime, you can use my age implementation with it: https://github.com/str4d/rage

**Reply by myyra** [reply]
Nice! Do you think your implementation will be compatible with the future official version?
